odin/apps/core/kafka.py

```
@@ -42,3 +43,39 @@ def send_message(cls, topic: str, message: dict[str, Any]) -> None:
+            for _ in range(100):
+                records = consumer.poll(timeout_ms=1000)
+                for _tp, messages in records.items():
+                    for message in reversed(messages):
+                        data = message.value
+                        if data.get("relay_id") == relay_id:
+                            state = data.get("state")
+                            consumer.close()
+                            return state
+
+            consumer.close()
+            return None
```

Ruff (0.15.0)

[warning] Consider moving this statement to an `else` block

(TRY300)